lib/modules/manager/buildpacks/index.ts

```diff
@@ -0,0 +1,12 @@
+import type { Category } from '../../../constants';
+import { DockerDatasource } from '../../datasource/docker';
+export { extractPackageFile } from './extract';
+
+export const defaultConfig = {
+  commitMessageTopic: 'buildpacks project descriptor {{depName}}',
+  fileMatch: ['(^|/)project\\.toml$'],
```

Comment:
Julia uses `Project.toml`[^1] but the different case shouldn't cause false positives

[^1]: https://pkgdocs.julialang.org/v1/toml-files/

Reply:
the `fileMatch` pattern won't match against `Project.toml`